plugins/thunderbolt/fu-thunderbolt-device.c

```diff
@@ -356,10 +356,9 @@ fu_thunderbolt_device_setup_controller(FuDevice *device, GError **error)
 	fu_device_set_name(device, tmp);
 
 	/* set vendor string */
-	tmp = fu_udev_device_get_sysfs_attr(FU_UDEV_DEVICE(self), "vendor_name", error);
-	if (tmp == NULL)
-		return FALSE;
-	fu_device_set_vendor(device, tmp);
+	tmp = fu_udev_device_get_sysfs_attr(FU_UDEV_DEVICE(self), "vendor_name", NULL);
+	if (tmp != NULL)
+		fu_device_set_vendor(device, tmp);
```

Comment:
Won't this be problematic for targeting firmware if we don't have a vendor ID set?  IIRC that's the reason it was mandatory.

Reply:
What I'm trying to accomplish is to have a host controller show up as a device.. It does not necessarily needs to be an updatable device. This is to enable the use case in https://github.com/fwupd/fwupd/issues/3356 where we would use the host controller to enable the children retimers for probing.

cc @RajmohanMani @krankun

Reply:
Isn't this just a string for information purposes? The vendor id is set a few lines below: 386/390

Reply:
Yup, Vendor is just for the UI. The vendor-id is the important one.